meshtastic/interfaces/ble/management_service.py

```
@@ -0,0 +1,596 @@
+                exc,
+                exc_info=True,
+            )
+            return ble_client_factory(target_address)
+        raise
+
+
+class BLEManagementCommandsService:
+    """Service helpers for BLE management command paths."""
+
+    @staticmethod
+    def execute_management_command(
```

Ruff (0.15.5)

[warning] Too many branches (14 > 12)

(PLR0912)